# Identifying and targeting abnormal mitochondrial localization associated with psychoses

Marzieh Haghighi1\*, Donna McPhie2\*, Mohammad Rohban1, Erin Weisbart1, David J. Logan1, Kyle W. Karhohs1, Jessica D. Ewald1, Johan Fredin Haslum1, Beth A. Cimini1, Shantanu Singh1, Bruce M. Cohen2^, Anne E. Carpenter1^

1. Imaging Platform, Broad Institute of MIT and Harvard, Cambridge MA USA 02142  
2. McLean Hospital, Harvard Medical School

\*equal contribution  
^ equal contribution, corresponding authors

## **Abstract**

Therapeutics working by novel mechanisms are needed for patients with psychiatric conditions. Cell-based assays to identify candidates that reverse observed abnormalities could accelerate the process. Here, we imaged peripheral cells (skin fibroblasts) of 168 patients, stained for DNA, actin, and mitochondria. We found mitochondria tend to be farther from the cell border for patients who experience psychosis (including subsets of individuals with bipolar disorder, schizophrenia, and schizoaffective disorder). We observed a reverse trend, albeit not statistically significant, for patients diagnosed with major depression. Because the mitochondrial dispersion phenotype could be identified by a single metric, we were able to readily query existing databases of cells stained for their mitochondria and treated with various chemical or genetic perturbations. We identified compounds and genes both negatively and positively affecting the psychosis-associated phenotype, including some known to impact psychiatric conditions. Developing therapeutics with novel mechanisms is a complex multi-step challenge. This cell-based assay holds promise for virtual and physical screening to identify candidates for treating psychiatric conditions.

## **Introduction**

Many patients presenting with psychosis respond only partially or poorly to current treatments. They are in critical need of medicines with novel mechanisms of action, both to improve efficacy and reduce disabling side effects relative to existing medicines [(Cohen & Öngür 2018; Stępnicki et al. 2018; McIntyre et al. 2020; Pereira et al. 2018; Kuperberg et al. 2021\)](https://paperpile.com/c/wQZsmN/QhnA+QvSH+X8Lz+OoYi+YTDY). These conditions include schizophrenia, schizoaffective disorder and bipolar disorder, which collectively have an enormous economic, social, and emotional impact [(Bessonova et al. 2020; Kadakia et al. 2022; Lin et al. 2022\)](https://paperpile.com/c/wQZsmN/L5pm+cCFQ+25Hj).  

Unlike many diseases, target-based drug discovery is not ideal for these conditions. Hundreds and even thousands of genes underlie the risk for psychotic disorders, and no single gene determines more than a modest proportion of risk [(Trubetskoy et al. 2022\)](https://paperpile.com/c/wQZsmN/3sbC). There is limited knowledge about the identity and function of all the proteins and other cellular components involved in the cellular processes that ultimately underlie and lead to psychosis. Current standard medications were discovered serendipitously. Predicting new protein or cellular targets whose chemical modulation will produce the desired antipsychotic effect in humans, based on existing understanding, is error-prone, making the unbiased approach of phenotypic assays appealing. However, ideal phenotypic assays involve a faithful, high-throughput model of the disease pathogenicity in a dish. This is inherently challenging for conditions involving the brain. Although complex brain organoids can be engineered [(Qian et al. 2019; Tan et al. 2021\)](https://paperpile.com/c/wQZsmN/mxO2+g4Wy), they are not well-suited to high throughput screening. More importantly, it is unclear what phenotypic effect ought to be sought in cells or organoids in order to identify compounds that relieve neuropsychiatric symptoms. Often, promising therapeutic hypotheses fail in the clinic given complexities at the organism level and a faulty link between a proposed target or phenotype and the human condition.

Rather than starting from a hypothesized phenotype based on existing scientific knowledge for a condition, recent evidence demonstrates that phenotypes can be identified in an unbiased way directly from patient samples, as we discuss next. Association of a cell phenotype with a condition does not prove causation; nevertheless, it enables drug screening, such that compounds reversing cell-based phenotypes are worth testing to see if they effectively reverse disease symptoms.

Patient fibroblasts, derived from a skin biopsy, are an appealing source material for identifying phenotypes because they are a cell type that is relatively accessible, acquirable from living patients, and sufficiently propagated for high-throughput screens. Although some fibroblasts share an origin with neuroectoderm, they would not be expected to have all of the necessary cellular pathways and structures to accurately reflect phenotypes for all human conditions. Still, in a promising set of findings, several neurological conditions were recently shown to exhibit detectable phenotypes in patient fibroblasts. Focusing on studies that revealed morphological phenotypes, mitochondrial morphology was found to be disrupted in fibroblasts from 41 patients with mid-stage idiopathic Parkinson’s disease as compared to controls [(Antony et al. 2020\)](https://paperpile.com/c/wQZsmN/QtHG) and a complex morphological change was found in 46 patients with Parkinson’s – 32 sporadic, 14 carrying particular mutations – compared to matched healthy controls [(Schiff et al. 2022\)](https://paperpile.com/c/wQZsmN/SDRm). Mitochondrial and nucleus staining of fibroblasts distinguished several genetic and chemical neuronal models of Parkinson’s disease from matched controls [(Thomas et al. 2023\)](https://paperpile.com/c/wQZsmN/09xQ). In spinal muscular atrophy, machine learning could distinguish fibroblast images of 12 patients from 12 healthy controls, though the study could not rule out technical confounders [(Yang et al. 2019\)](https://paperpile.com/c/wQZsmN/D1c9). SORL1 is a gene associated with Alzheimer’s disease; morphology phenotypes distinguished SORL1 knockout neural progenitor cells from isogenic wild-type controls [(McDiarmid et al. 2023\)](https://paperpile.com/c/wQZsmN/ARxg). 15 people with two genotypes of hereditary spastic paraplegia were distinguished from matched controls using stains for DNA, mitochondria, and acetylated α-tubulin, plus label-free phase contrast cell images in fibroblasts [(Wali et al. 2021\)](https://paperpile.com/c/wQZsmN/U7BT). Other studies also uncovered phenotypes for neurological conditions in smaller cohorts [(Blanchet et al. 2015; Hung et al. 2018; Wali et al. 2023\)](https://paperpile.com/c/wQZsmN/J0DX+koRS+DGzT).

Therefore, although some conditions might only have phenotypes detectable at the organ or organismal level, others do seem to manifest at the peripheral cell level. Similarly, although some disease-related phenotypes might only be visible in a “relevant” cell type, others may be detectable in a broader spectrum of cell types, including fibroblasts. This may be due to the repurposing of many cellular components across diverse functions in different cell type contexts, or because cellular mechanisms may exist and be impacted in a diversity of cell types even though the symptoms experienced by a patient are only observed in a particular organ system, perhaps due to compensatory mechanisms elsewhere. For psychotic disorders, it is notable that patients are at elevated risk for other non-psychiatric disorders and there are substantial overlaps between gene sets associated with the two [(Bulik-Sullivan et al. 2015; Andreassen et al. 2013\)](https://paperpile.com/c/wQZsmN/LtTU+Gkwf). Therefore, it would not be surprising if peripheral cells showed abnormalities relevant to those in brain cells. Because obtaining live cells from the human brain is rarely possible and because generating relevant neural cell types by either differentiation of hiPSC lines or by direct conversion of somatic cells is challenging at scale [(Panchision 2016\)](https://paperpile.com/c/wQZsmN/3hIX), identifying phenotypes in cell types that are readily obtained and high-throughput-compatible could accelerate discovery.

In 2010, we reported finding a morphological phenotype in fibroblasts and lymphocytes from 10 bipolar patients as compared to 10 control patients [(Cataldo et al. 2010\)](https://paperpile.com/c/wQZsmN/xSLK). Specifically, the distribution of mitochondria was unusual in bipolar patients, whose cells displayed mitochondrial networks that were much more dense and concentrated near the nucleus, versus controls whose mitochondria dispersed more evenly throughout the cell.

The finding of morphological abnormalities in mitochondria of patients with conditions producing psychosis is now affirmed by a wide variety of genetic, brain imaging, and peripheral cell studies [(Scaini et al. 2021; Flippo & Strack 2017; Papageorgiou & Filiou 2024\)](https://paperpile.com/c/wQZsmN/mt7A+Z0wI+PI3F). Postmortem brains of bipolar patients have more, but smaller, mitochondria [(Cataldo et al. 2010\)](https://paperpile.com/c/wQZsmN/xSLK), as do their fibroblasts *ex vivo* [(Marques et al. 2021\)](https://paperpile.com/c/wQZsmN/h6iR) and iPSC-derived neurons, *in vitro* [(Mertens et al. 2015\)](https://paperpile.com/c/wQZsmN/CZxl). Mitochondrial fragmentation is also seen in Epstein-Barr virus (EBV)-transformed lymphocytes from schizophrenic patients [(Rosenfeld et al. 2011\)](https://paperpile.com/c/wQZsmN/XeN6). Similar glutamate-induced fragmentation can be mitigated in mouse hippocampal HT22 cells when the psychiatric disorder-associated gene CACNA1C [(Moon et al. 2018; Sklar et al. 2008\)](https://paperpile.com/c/wQZsmN/XCR9+aijc) is knocked down by RNA interference [(Michels et al. 2018\)](https://paperpile.com/c/wQZsmN/Boan). Postmortem brains of patients with schizophrenia show a lower density of mitochondria in axon terminals, compared to healthy brains [(Roberts 2021\)](https://paperpile.com/c/wQZsmN/2MTZ). Evidence for functional abnormalities in mitochondria is even more solidly associated with these conditions, ranging from genetics [(Cuperfain et al. 2018\)](https://paperpile.com/c/wQZsmN/TvoK), alterations in mitochondrial fusion/fission proteins [(Scaini et al. 2017\)](https://paperpile.com/c/wQZsmN/nsl6), bioenergetics [(Zanfardino et al. 2021\)](https://paperpile.com/c/wQZsmN/slO6), and more [(Scaini et al. 2021; Wu et al. 2019; Scaini et al. 2016; Pereira et al. 2018; Kuperberg et al. 2021; Giménez-Palomo et al. 2021\)](https://paperpile.com/c/wQZsmN/mt7A+hwWJ+LS2g+OoYi+YTDY+UDFu). Likewise there is evidence for morphological and functional impairment of mitochondria in major depressive disorder [(Culmsee et al. 2018; H. Chen et al. 2024\)](https://paperpile.com/c/wQZsmN/od56+SUKK). Collectively, these findings are consistent with the more general conclusion that mitochondrial morphology and function are tightly related and abnormal in many psychiatric conditions [(Chan 2020; Bulthuis et al. 2019; Jenkins et al. 2024; Ni et al. 2024\)](https://paperpile.com/c/wQZsmN/iAYU+AnbX+DbQN+kJ46).  

Here, we sought to increase confidence in a mitochondrial localization phenotype in psychiatric conditions by testing a new, larger cohort: 168 people as compared to our original set of 20\. Indeed, the lack of attempts to replicate findings in larger cohorts has been described as limiting progress in the field [(Harrison et al. 2020\)](https://paperpile.com/c/wQZsmN/drqs). We also sought to expand the study to related psychiatric conditions beyond bipolar disorder \- for the past fifteen years, we have been collecting samples from patients experiencing psychosis across a spectrum of conditions including schizophrenia, schizoaffective disorder and bipolar disorder, along with healthy control patients and patients with major depressive disorder. Finally, because computational analysis of images has advanced dramatically in the past decade and public datasets are now available, we were also able to develop and pilot a virtual screening approach based on mitochondrial localization that might identify chemical and genetic perturbations relevant to the psychiatric conditions in other datasets.

## **Results**

## ***Mitochondrial localization is disrupted in fibroblasts from patients experiencing psychosis***

We sought to identify a morphological phenotype in a cohort of patients experiencing psychosis. Such a phenotype could subsequently be used to screen public image datasets to identify potential therapeutics; that is, agents that impact the illness-related phenotype (Figure 1). We collected skin punch biopsies from patients and derived human fibroblasts from them, including from 50 people with a primary diagnosis of bipolar disorder (BP), 39 with schizophrenia (SZ), 11 with schizoaffective disorder (SZA), 21 with major depressive disorder (MDD), and 47 healthy controls; together with patient demographic metadata including their primary diagnosis. The median and range of ages was consistent across patient categories, as was the typical male-skewed ratio seen for these conditions, except for MDD (Supplementary Figure 1). We chose fibroblasts for their benefits described in the introduction; furthermore, schizophrenia and bipolar disorder have known associations with skin abnormalities [(McPhie et al. 2021\)](https://paperpile.com/c/wQZsmN/YNXI), serving as additional rationale that this cell type may exhibit disease-associated phenotypes. We grew the cell lines as consistently as possible prior to plating, fixing, staining them for DNA, actin, and mitochondria (using MitoTracker-M7510), and imaging them (Figure 2). By eye, we observed that samples from patients in the control and MDD categories show a more fine-grained, dispersed mitochondrial network extending to the edges of the cell, whereas patients in the categories experiencing psychosis tend to show an agglomerated, thicker network more concentrated around the nucleus. The pattern is subtle and heterogeneous across a cell population. We next sought metrics to quantify these observations objectively, consistently, and efficiently, hoping we could use the resulting phenotype for screening existing public datasets.

**![][image1]**  
***Figure 1:** **Overview of our strategy for phenotype discovery and virtual screening.***

*![][image2]*  
***Figure 2: Morphological phenotypes associated with psychiatric conditions are visible but subtle.** Fibroblasts that are stained for DNA, actin, and mitochondria, and labeled according to the patient category. The representative images are from the patient with the median value of the feature Cells\_RadialDistribution\_MeanFrac\_mito\_tubeness16of16 for that class (see Figure 4c). Scale bar \= 20 μm for the first four images in each row, which are full fields of view. Cropped images in the last column reflect 30% of each field of view in the experiment. Images are clipped to their 0.5 and 99.5 percentile of their range and scaled to \[0, 255\] range just for visual clarity, given that the overall intensity of the staining is not a phenotype under study.*

To preserve interpretability, we used classical image processing techniques to extract well-understood features of morphology, which include measures of size, shape, intensity and texture of the cell components observed using the various stains [(Stirling et al. 2021\)](https://paperpile.com/c/wQZsmN/uQ4D). Machine learning classifiers were able to distinguish BP, SZ, and SZA patient groups from healthy controls using linear regression; the combination of these three groups who all experience psychosis was also distinguishable  (Figure 3). The strongest phenotypes were seen in patients with the affective conditions, BP and SZA. This may be consistent with the observation that such patients, especially those in a manic episode, have the most extreme episodes of psychosis with so-called ‘positive’ symptoms, observed as hallucinations, delusions and agitated behavior [(Carlson & Goodwin 1973\)](https://paperpile.com/c/wQZsmN/PqA2). A convolutional neural network based on raw images’ pixels yielded roughly similar results (Supplementary Figure 2). By contrast, the MDD patient group was the least distinguishable from healthy controls. Unlike diseases whose phenotypes readily and completely separate patients from controls, we were not surprised that phenotypes for these mental conditions were subtle: most patient groups were not completely distinguishable from healthy controls in all folds of our 10-fold cross validation run. This is expected given genetic, diagnostic, and environmental variability among patients, as well as the fact that dramatic changes in cell phenotypes would likely be associated with death in utero or early and substantial cognitive impairments such as those seen in neurodevelopmental disorders. We believe that risk of illness is likely determined by more than mitochondrial distribution alone, yet this phenotype could be useful for drug screening.

![][image3]

***Figure 3: Machine learning classification based on multiple mitochondrial features distinguishes patient group experiencing psychosis from healthy controls**. Logistic Regression (LR) classifiers were trained on patient-level profiles constructed from mitochondrial features extracted using CellProfiler (CP) software (CP-LR). Patient categories include those with a primary diagnosis of bipolar disorder (BP), schizophrenia (SZ), schizoaffective disorder (SZA), and major depressive disorder (MDD). Psychosis encompasses patients in the BP, SZ, and SZA groups. Classifier performance is evaluated using 10-fold cross-validation, with the distribution of the area under the receiver operating characteristic curve (AUC-ROC) for each binary classification shown as individual data points on the box plots. The box plots represent the interquartile range (25th–75th percentile), with the median marked by a horizontal line. The null distribution (blue) was generated by training classifiers on shuffled patient class labels. Statistically significant differences between classifications (e.g., psychosis vs. healthy) are denoted by asterisks. The cellular features captured here are most prominent in participants with affective psychoses, BP and SZA.*

We have been pioneers in the use of machine learning, including deep learning, to detect morphological phenotypes in images, and others have used machine learning to distinguish disease-associated samples (see Introduction). These strategies leverage hundreds to thousands of features. However, it is preferable for screening purposes for the phenotype to be a single feature. A single feature (or a combination of a few) is more readily translatable across cell types and experimental batches, including existing publicly available datasets. Keeping the number of features low is also valuable to conserve statistical power and avoid overfitting that can occur when measuring thousands of morphological features for only dozens of samples [(Lones 2021\)](https://paperpile.com/c/wQZsmN/tVws).

We therefore sought a simpler morphological metric for the phenotype associated with these psychosis-related conditions of interest. We designed a metric called MITO-SLOPE to capture the phenotype (Figure 4, with details in Methods) based on the data, our prior study [(Cataldo et al. 2010\)](https://paperpile.com/c/wQZsmN/xSLK), knowledge of these conditions and the role of mitochondria in them, and of morphological cell image analysis. MITO-SLOPE quantifies the gradient of change in mitochondrial network density at the cell periphery and is based on a set of related CellProfiler features (Cells\_RadialDistribution\_MeanFrac\_mito\_tubenessNNof12) that measure the mean fraction of the MitoTracker (mitochondrial) staining intensity in 12 concentric rings, starting at the edge of the nucleus and ending at the cell border.

![][image4]  
***Figure 4: Localization of mitochondria differs in various patient groups experiencing psychiatric conditions**. (a) Schematic illustrating how radial features are computed for MitoTracker staining using concentric rings from the nucleus to the cell border (4 bins shown for illustration; 12 bins are actually used in this study). MITO-SLOPE is defined as the slope of this feature pattern at the cell periphery (see Methods for definition), capturing differences in mitochondrial enrichment or depletion across groups. (b) Mean standardized values (± standard error) of 12 radial mitochondrial features (Cells\_RadialDistribution\_MeanFrac\_mito\_tubenessNNof12 where NN is the bin number), ordered from the edge of the nucleus (1of12) to the cell periphery (12of12), and standardized to the control group. Patients with bipolar disorder (BP, dark red) and schizophrenia (SZ, red) show a declining pattern at the cell periphery, indicating reduced mitochondrial density near the cell edge. In contrast, major depressive disorder (MDD, blue) shows increased peripheral mitochondrial signal. (c) The value of the MITO-SLOPE feature representing mitochondrial dispersion is shown for each patient (derived from the mean-averaged pattern across the cells within each patient) in each category. “Psychosis” includes the patients in the categories BP, SZ, and SZA. Box plots show the median and  upper and lower quartiles and whiskers denote the 1.5× interquartile range. P-values are noted for each comparison to the control group.*

For all patient groups, mitochondrial staining is higher at the edge of the nucleus and decreases towards the cell edge \- however, the *relative* staining intensity compared to controls, as measured by MITO-SLOPE, differs. Specifically, MITO-SLOPE is decreased (relative to healthy controls) for patients who experience psychosis (p \= 0.0069), particularly the subsets of that group with bipolar disorder (p \= 0.0061) and schizophrenia (p \= 0.017) (Figure 4c). This means their mitochondria are localized closer to the nucleus and exhibit a decline at cell edges. MITO-SLOPE for the schizoaffective disorder subset of patients shows a trend in the same direction but with a p value of 0.56; this category had the smallest number of patients (11). Patients with depression show a non-statistically-significant trend (p \= 0.18 \- the second smallest category, with 21 patients) in the opposite direction, with mitochondria showing a relative increase at the edge of the cell. Again, we note that individual patients’ values for the phenotype overlap the healthy controls and each other; this is not surprising given that environmental factors contribute to risk and given that these diagnoses are not as dramatically distinctive as, for example, the other neurological disorders mentioned in the introduction, where in some cases only a dozen samples were sufficient to distinguish the condition.

These findings are consistent with our prior published results on 10 bipolar patients [(Cataldo et al. 2010\)](https://paperpile.com/c/wQZsmN/xSLK) but expands to a larger number and includes other conditions that present with psychosis, such as schizophrenia. It is also consistent with our prior work that found a mitochondrial localization phenotype in patient astrocytes for a schizophrenia-predisposing syndrome associated with  22q11.2 deletion [(Tegtmeyer et al. 2024\)](https://paperpile.com/c/wQZsmN/JVpP).

## ***Chemical regulators of mitochondrial localization***

Large public image datasets are becoming available, enabling an intriguing possibility: chemical regulators of relevant pathways, and potential therapeutics, might be identified by assessing disease-associated phenotypes in existing datasets of compounds. Virtually screening existing datasets saves significant cost and time relative to setting up and conducting physical experiments, and such screening has proven effective in identifying promising chemical regulators [(Rohban et al. 2022\)](https://paperpile.com/c/wQZsmN/zbek) and therapeutic candidates [(Cuccarese et al. 2020\)](https://paperpile.com/c/wQZsmN/QI5S). There are caveats. The approach might not work well if the available data are from a different cell type than the cell type where a disease phenotype was identified. However, our ability to find a psychosis-associated phenotype in skin cells, for patients experiencing a neurological anomaly, provides a hint that this phenotype may be present in many cell types. A virtual screen could also fail if effective chemicals are missed because they would only reverse a phenotype in patients’ cells that show the phenotype and not show any impact on cells from people not having a psychotic disorder (as is likely the case for most cultured cell lines used in public databases).

Image-based screens, where compounds are physically tested for reversion of a disease-associated phenotype of interest, are very common. However, only recently has image-based *virtual* (or *inferential*) screening been demonstrated [(Rohban et al. 2022; Anon 2021; Celik et al. 2024; Anon n.d.)](https://paperpile.com/c/wQZsmN/zbek+Xrlx+XEAx+BpQy). To our knowledge, a large image-based virtual screen based on a phenotype uncovered in patient fibroblasts has not been attempted. The strategy is analogous to connectivity mapping using transcriptional profiles. For example, a recent study reported using post-mortem transcriptional signatures to virtually query the large Connectivity Map gene expression database to identify 35 drugs that target genes associated with the shared risk pathways across BP and SZ, including some that interact with CACNA1D [(Grotzinger et al. 2023\)](https://paperpile.com/c/wQZsmN/t3cl).

We therefore queried existing Cell Painting datasets. We began with the LINCS dataset cpg0004-lincs [(Way et al. 2022\)](https://paperpile.com/c/wQZsmN/tjlZ) because it includes mostly approved drugs, allowing us to analyze drug categories known to treat certain symptoms for patients with schizophrenia and bipolar disorder. A colleague manually annotated such drugs into various categories based on their mechanisms (Dr. Rakesh Karmacharya, see Acknowledgements), and we found that none of the known psychoactive categories we annotated were statistically enriched in either direction in our ranked list of phenotype strength (Figure 5). This result is consistent with our previous finding that mitochondrial localization in fibroblasts was not impacted by lithium treatment, nor did patients’ lithium use alter mitochondrial morphology in postmortem brain samples [(Cataldo et al. 2010\)](https://paperpile.com/c/wQZsmN/xSLK). We note that existing drugs typically do not clearly and specifically target a single protein or process, so these annotated categorizations are imperfect. Further, fibroblasts do not likely express high levels of the proteins that are the direct targets of most of these psychiatric drugs, nor are fibroblasts part of circuits equivalent to the neural networks that mediate the clinical effects of these drugs. In addition, the therapeutic effects of drugs used for psychosis and depression develop over weeks of treatment. Nevertheless, this affirms that the assay has the potential to identify candidate therapeutics working by a novel mechanism of action that does not clearly map to existing drug categories.

![][image5]

***Figure 5: Existing classes of psychoactive drugs do not, overall, show evidence of impacting mitochondrial localization.** For drug mechanism-of-action classes with more than four compounds present in the dataset, we assessed their impact on the mitochondrial localization phenotype, as measured by the mean value of MITO-SLOPE feature across all cells in the sample. None of the mechanism-of-action classes shows a statistically significant rank ordering (in either direction) relative to the other compounds in the assay. NES: normalized enrichment score (red indicates positive NES: more towards the top of the list of compounds rank-ordered by MITO-SLOPE from positive to negative; blue indicates the class is enriched at the negative end of the list \- though, again, none of the categories was statistically significantly enriched); FDR: false discovery rate from Benjamini-Hochberg multiple hypothesis correction across all eight drug classes (see Methods). To avoid loss of signal from samples treated with very low concentrations of each drug, we selected (for each compound) the dose that yielded the strongest impact on this phenotype. Results for all classes of compounds and all doses independently are shown in Supplementary Figure 4\. All data were cell-count-corrected.*

We then virtually screened the LINCS dataset to find “hits”, drugs that impact MITO-SLOPE strongly in either direction. We quickly noticed that many cancer drugs yield strong changes in MITO-SLOPE but also change many other features of cells that would likely lead to adverse outcomes, and would therefore be undesirable for a drug to treat a chronic mental health condition. We therefore designed a strategy to filter hits to only keep those that minimally change morphology of cells within the intersection of two feature subspaces: (1) features that are orthogonal (do not correlate) to mitochondrial localization, and (2) features that are not distinctive across patient groups in the fibroblast dataset (see Methods).

We found seven positive MITO-SLOPE hits (with potential to revert the psychosis phenotype) and three negative hits (with potential to revert a depression phenotype, if indeed the conditions are at opposite ends of a mitochondrial-localization spectrum) (Figure 6).

![][image6]

***Figure 6: Chemical hits in virtual screen impact mitochondrial localization while minimally impacting cell morphology overall**. (a) The table shows compounds that increase (blue) or decrease (red) mitochondria towards the edge of the cells as measured by d\_slope (d\_slope is the Cohen’s d of the t-test comparing the negative control versus drug-treated cell population, standardized for sample size; the directionality matches the mean of the original values MITO-SLOPE, and the value matches the strength MITO-SLOPE). p\_orth\_std measures whether non-mitochondrial-related features were significantly different from controls; a filter of 0.05 (after adjusting for multi-hypothesis testing) was applied to ensure the remainder of cell morphology remained fairly normal (see Methods). Of note for drug toxicity, the hits do not dramatically impact cell count, within the time frame of the experiment. Values were calculated for each compound’s dose independently. Annotated mechanisms are from the Drug Repurposing Hub. (b and c) Representative images of selected hits from the LINCS dataset showing A549 cells treated with (b) 0.04 μm divalproex-sodium showing mitochondria spread throughout the cytoplasm, including the cell edge,  and (c) 3.33 μm PF-02545920 showing mitochondria condensed towards the nucleus. White outlines of cell borders are generated during image analysis; blue \= DNA, red \= MitoTracker. Image intensities were rescaled relatively for each image (using the minimum/maximum of each image) for better visualization of the staining pattern. See Methods for image scaling parameters. Scale bars \= 25 micrometers.*

Among the hits, a few drugs are used for psychiatric or neuropsychiatric conditions. Citalopram is a selective serotonin reuptake inhibitor (SSRI) used to treat depression [(Sharbaf Shoar et al. 2025\)](https://paperpile.com/c/wQZsmN/Ax6j); it shows a negative MITO-SLOPE which is the direction expected to normalize the dispersed mitochondrial localization associated with depression. Divalproex-sodium is a benzodiazepine receptor agonist and HDAC inhibitor [(Rahman et al. 2025\)](https://paperpile.com/c/wQZsmN/8a1N) used to manage a variety of seizure disorders [(Willmore 2003\)](https://paperpile.com/c/wQZsmN/70Wd) and bipolar disorder [(Bond et al. 2010; Cipriani et al. 2013\)](https://paperpile.com/c/wQZsmN/5HR7+RdVx); it shows a positive MITO-SLOPE which is the direction expected to normalize the centralized mitochondrial localization associated with psychosis. Tolcapone, a Parkinson’s disease medication, is a catechol-O-methyltransferase (COMT) inhibitor that affects dopamine levels, which is strongly linked to many psychiatric conditions. Its side effects can include hallucinations and depression [(Willman & Tadi 2025\)](https://paperpile.com/c/wQZsmN/7dVa).

Two drugs are known to interact with the phosphodiesterase (PDE) protein family, which has been strongly linked to psychiatric disorders [(Hebb & Robertson 2007; Jiang et al. 2023\)](https://paperpile.com/c/wQZsmN/LbgM+x34r) and whose inhibitors have received trials in mood and psychotic disorders, with mixed results [(Delhaye & Bardoni 2021; Sadeghi et al. 2023\)](https://paperpile.com/c/wQZsmN/jAQ6+Hnps): PF-02545920 is a PDE10a inhibitor that did not progress after Phase 2 trials in Huntington’s disease [(Ferguson et al. 2022\)](https://paperpile.com/c/wQZsmN/aK9i) nor Phase 1b/2 trials in schizophrenia [(DeMartinis et al. 2019; Walling et al. 2019\)](https://paperpile.com/c/wQZsmN/xWJS+zEyQ), although another PDE10a inhibitor passed Phase 1 trials in Tourette syndrome [(Marshall et al. 2024\)](https://paperpile.com/c/wQZsmN/QsEl). Trapidil is PDE3 inhibitor and competitive inhibitor of the platelet-derived growth factor receptor (PDGFR) inhibitor that functions as a vasodilator and is subject to a patent for cognitive disorders including depression and psychosis [(Bordbar 2022\)](https://paperpile.com/c/wQZsmN/1b7x). Note that these PDE inhibitors have the opposite impact on mitochondrial localization; different PDEs are believed to have different impacts on psychiatric function: as noted in [(Sadeghi et al. 2023\)](https://paperpile.com/c/wQZsmN/Hnps), “The most promising therapeutic candidates to emerge from these preclinical studies are PDE2 and PDE4 inhibitors for depression and anxiety and PDE1 and PDE10 inhibitors for schizophrenia. Furthermore, PDE3 and 4 inhibitors have shown promising results in clinical trials in patients with depression and schizophrenia”. Caffeine also targets PDEs but its effect did not reach statistical significance (d\_slope \= \-0.45).

The MITO-SLOPE hits also included drugs without a history in psychiatric treatment, but that act by plausible mechanisms and may warrant further exploration. For example, XL-147 inhibits PI3K, a pathway considered as a target for psychiatric disorders [(Y. Chen et al. 2024\)](https://paperpile.com/c/wQZsmN/ubQl). Tivozanib is an inhibitor of the vascular endothelial growth factor receptor (VEGFR), a pathway with ties both to schizophrenia [(Rampino et al. 2021\)](https://paperpile.com/c/wQZsmN/dAwW) and depression [(Wang et al. 2023\)](https://paperpile.com/c/wQZsmN/QZLl). Clioquinol is an antifungal with a poorly understood mechanism of action [(McKeny et al. 2025\)](https://paperpile.com/c/wQZsmN/iCDw), removed from most markets due to an association with Subacute Myelo-Optic Neuropathy [(Bareggi & Cornelli 2012\)](https://paperpile.com/c/wQZsmN/stei). It has shown preclinical promise in animal studies of Alzheimer's disease [(Adlard et al. 2008\)](https://paperpile.com/c/wQZsmN/RjGq).

We analyzed two other public imaging datasets where compounds’ impacts on mitochondrial localization could be discerned. These compound sets have far fewer annotated mechanisms of action, because they primarily contain screening compounds rather than well-studied drugs or bioactive agents. Nevertheless, we provide both lists for future studies (see Data and Code Availability). For example, these compounds could be used to train machine learning models to predict mitochondrial localization from chemical structures. In the cpg0016-jump (JUMP) dataset, we found 228 compounds meeting both criteria: statistically significant MITO-SLOPE values and fairly normal cell morphology based on the orthogonal features, but none had annotated mechanisms of action. In the cpg0012-wawer-bioactivecompoundprofiling (CDRP) dataset, we found 142 compounds as hits. Of these, only two had annotations: BRD-K28863208 (also known as PNU-282987; d\_slope \= \-0.37), an agonist of the nicotinic cholinergic receptor encoded by *CHRNA7,* and BRD-K85242180 (-0.56), an indoleamine 2,3-dioxygenase (IDO) inhibitor also connected to nicotinic cholinergic receptors: IDO is a key enzyme in the kynurenine metabolic pathway, which has been observed to be dysregulated in both SZ and BD [(Erhardt et al. 2017\)](https://paperpile.com/c/wQZsmN/Ka8E), and kynurenic acid modulates glutamatergic and dopaminergic neurotransmission as well as antagonizing alpha7 nicotinic cholinergic receptors [(Erhardt et al. 2017; Plitman et al. 2017\)](https://paperpile.com/c/wQZsmN/Ka8E+frcn). Genetic associations have been observed between alpha nicotinic receptors and both SZ and BD [(Thomsen et al. 2011\)](https://paperpile.com/c/wQZsmN/bk3u). Smoking cigarettes, a nicotine delivery system, is common in both SZ and BD [(Heffner et al. 2011; Ding & Hu 2021\)](https://paperpile.com/c/wQZsmN/Iz1P+TFRA); nicotine is known to regulate numerous neurotransmitters and can both reduce and exacerbate mood and cognitive symptoms [(Ripoll et al. 2004; Heffner et al. 2011; Ding & Hu 2021\)](https://paperpile.com/c/wQZsmN/B14g+Iz1P+TFRA). PNU-282987 was found to reverse an amphetamine-induced gating deficit in rats [(Bodnar et al. 2005\)](https://paperpile.com/c/wQZsmN/22jC) and is listed as a preclinical program for schizophrenia from Pfizer, terminated for undisclosed reasons [(Gray & Roth 2007\)](https://paperpile.com/c/wQZsmN/gygE). The characteristics of these two compound hits provide some support that they produce effects relevant to the mechanisms underlying SZ or BD.

## ***Genetic regulators of mitochondrial localization***

We next virtually screened our public JUMP Cell Painting images from genetic perturbations, both overexpression (12,609 genes) and CRISPR-Cas9-mediated knockout (7,975 genes) [(Chandrasekaran et al. 2024\)](https://paperpile.com/c/wQZsmN/aU1B). Because no single gene determines more than a small fraction of risk for SZ or BD, we did not anticipate dramatic impacts on the mitochondrial localization phenotype, but we hoped this analysis might provide some insights and further validate the mitochondrial localization phenotype as being relevant to bipolar disorder and schizophrenia.

First, we found that CRISPR-Cas9-mediated transient knockout of a group of genes annotated as impacting mitochondrial dynamics (fusion, fission, transport, and mitophagy) [(Cuperfain et al. 2018\)](https://paperpile.com/c/wQZsmN/TvoK) was statistically significantly enriched as lowering MITO-SLOPE (Figure 7a, less mito staining at the edge of the cell, mimicking the psychosis patient samples). The narrower mitochondrial transport category did not reach statistical significance but showed the same trend. We did not see enrichment for overexpressed genes in the same categories (Figure 7b), perhaps to be expected given that overexpression of a gene does not have a predictable impact on its function (more protein does not guarantee more of that protein’s function) and generally produces less robust phenotypes in the JUMP dataset in our previous analysis [(Chandrasekaran et al. 2024\)](https://paperpile.com/c/wQZsmN/aU1B).

Next, we examined two particular genes associated with schizophrenia and/or bipolar disorder: The association of CACNA1C with psychotic and mood disorders is a well-replicated finding [(Moon et al. 2018; Sklar et al. 2008\)](https://paperpile.com/c/wQZsmN/XCR9+aijc) and GSK3B is a known target for bipolar medicines, both antipsychotics [(Sutton & Rushlow 2011\)](https://paperpile.com/c/wQZsmN/b5Gf) and lithium [(Snitow et al. 2021\)](https://paperpile.com/c/wQZsmN/QjaA), with some variants associated with psychosis in some populations [(Jope & Roh 2006\)](https://paperpile.com/c/wQZsmN/NGhr) (though, as with most genes, it explains little of risk). We found that CRISPR knockout of CACNA1C had a dramatic pattern, matching that of major depression, while GSK3A and GSK3B had a much weaker impact (Figure 7c). We analyzed images from a public dataset from Recursion, RXRX3 [(Fay et al. 2023\)](https://paperpile.com/c/wQZsmN/uyqT), and confirmed the CACNA1C result while also showing a stronger result for both GSK3A and GSK3B (Figure 7d). GSK3B has a well-known role in mitochondrial activity (as reviewed in [(Yang et al. 2017\)](https://paperpile.com/c/wQZsmN/lQM1)), while CACNA1C has emerging evidence [(Theeuwes et al. 2018; Kessi et al. 2023; Michels et al. 2018\)](https://paperpile.com/c/wQZsmN/4MxE+JlMC+Boan).

![][image7]  
***Figure 7: Mitochondrial localization impact for genes with known association with mitochondrial function or psychosis.** Mitochondrial dynamic genes are enriched for unusual MITO-SLOPE, for CRISPR knockouts (a) but not overexpressed genes (b); mitochondrial transport genes are not significantly enriched in either dataset. Gene annotations are taken from Cuperfain, et al., Table 3 [(Cuperfain et al. 2018\)](https://paperpile.com/c/wQZsmN/TvoK). FDR and NES are as defined in Figure 5\. (c and d): As calculated from images in the JUMP-CRISPR (c) or RXRX3-CRISPR (d) datasets, we show the distribution of mitochondria in bins from the nucleus (left) to cell edge (right), as in Figure 4b (which shows the patterns of each patient category; all 3 genes here show a similar pattern as seen in patients with major depression). Right: Example images from each gene alongside negative controls; an image was selected at random for each condition and cropped to regions with similar cell density representative of the whole image. Scalebar \= 50 uM.*

Finally, we analyzed the full list of gene perturbations in both JUMP datasets, to identify those that most strongly produced the psychosis-associated mitochondrial phenotype (Figure 8). Three genes met the criteria for their CRISPR-Cas9-mediated knockout impacting MITO-SLOPE without markedly impacting non-mitochondrial-related features: *SP1*, *TBX3*, and *KCNH6* (Figure 8a; CACNA1C was at the \~4th percentile and impacted other features). SP1 is a transcription factor regulating genes involved in many cellular processes, including mitochondrial transcription factors A and B (TFAM, TFB1M and TFB2M) in neurons [(Dhar et al. 2013\)](https://paperpile.com/c/wQZsmN/9st1). SP1 has a strong connection with schizophrenia as reviewed in the introduction of Pinacho, et al., which itself found that *SP1* mRNA expression is significantly increased in the hippocampus in patients with chronic schizophrenia in post-mortem samples [(Pinacho et al. 2014\)](https://paperpile.com/c/wQZsmN/39NK). SP1 is also associated with autism [(Thanseem et al. 2012\)](https://paperpile.com/c/wQZsmN/OM81) and Alzheimer’s disease [(Citron et al. 2008; Santpere et al. 2006\)](https://paperpile.com/c/wQZsmN/eBUC+IOGa). Particularly interesting is that SP1 regulates the expression of *TBX3*, the second-most significant gene on the list [(Smith et al. 2011\)](https://paperpile.com/c/wQZsmN/xqMY). TBX3 is another transcription factor, regulating developmental processes including in hypothalamic neurons [(Shi et al. 2022\)](https://paperpile.com/c/wQZsmN/tgK0), but without strong links to psychiatric illness. The third hit, *KCNH6*, is a potassium channel subunit that modulates cardiac repolarization and heart rate, as well as neuronal excitability, with some links to mitochondrial health. For example, it “enhances hepatic glucose metabolism by regulating mitochondrial Ca2+ levels and inhibiting oxidative stress” [(Zhang et al. 2022\)](https://paperpile.com/c/wQZsmN/Iz6z) and is located in mitochondria and impacts mitochondrial metabolism [(Han et al. 2025\)](https://paperpile.com/c/wQZsmN/nx3w) and function [(Zhao et al. 2023\)](https://paperpile.com/c/wQZsmN/aGwm). Most of the seven genes just below the threshold of significance (RHOU, POLE3, DRD1, KCNA1, FARS2, FOXQ1, ASCL1) have roles in mitochondrial function or psychiatric conditions as well.

![][image8]  
***Figure 8: Exploration of genetic hits in virtual screen.** The tables show genes that increase (blue) or decrease (red) mitochondria towards the edge of the cells as measured by d\_slope (as described in Figure 6\) in the two datasets (a) JUMP-CRISPR and (b) JUMP-overexpression. p\_orth\_std is used as an additional filter, measuring whether non-mitochondrial-related features were significantly different from controls \- to ensure the remainder of cell morphology remained fairly normal (see Methods). The hits do not dramatically impact cell count, as shown.*

The 39 hits in the overexpression data were not all clearly linked to mitochondrial function or psychiatric conditions; many are linked to cancer, indicating that they may be more related to changes in overall cell shape, despite the filter we applied to exclude major changes in orthogonal features. Enrichment analysis failed to find consistent known annotations among the hits, apart from some growth and cancer related categories around p \~0.05 (Supplementary Figure 3). Nevertheless, among the 14 negative slope hits, we found DELE1, a mitochondrial-localized protein involved in mitochondrial stress response [(Fessler et al. 2020\)](https://paperpile.com/c/wQZsmN/PDkn), which is itself linked to mood and psychotic disorders (see Introduction). Two genes relate to intracellular transport: DCTN4 (Dynactin Subunit 4\) is “part of the dynactin complex that activates the molecular motor dynein for ultra-processive transport along microtubules” [(GeneCards Human Gene Database n.d.)](https://paperpile.com/c/wQZsmN/b50J) and MARK3 (microtubule affinity-regulating kinase 3\) regulates microtubule stability [(Drewes et al. 1997\)](https://paperpile.com/c/wQZsmN/osPp). These functions can impact proper trafficking of mitochondria in axons, which is also linked to CNS disorders [(Zheng et al. 2019\)](https://paperpile.com/c/wQZsmN/nLXw); for instance, inhibiting dynactin components genetically in Drosophila yielded defective axonal transport of mitochondria [(Pilling et al. 2006; Saxton & Hollenbeck 2012\)](https://paperpile.com/c/wQZsmN/Pnwa+Dd1l). EDA2R was found as a biomarker associated with lower cognitive ability and brain volume in human aging [(Harris et al. 2020\)](https://paperpile.com/c/wQZsmN/ZtD8). MFAP3 has been reported as a biomarker for pain [(Niculescu et al. 2019\)](https://paperpile.com/c/wQZsmN/R4VB). The 10 other genes (CDK18, SHC1, IHH, DNAJC22, ATOH7, SCAMP4, PNLIP, KLHDC3, PSG2) have little direct linkage to mitochondrial positioning or psychiatric conditions, though some are mitochondria-localized or relating to membrane trafficking.

The 25 positive-slope JUMP-ORF genes also include cancer-related genes regulating growth and differentiation. Still, there are many other genes related to psychiatric conditions and/or mitochondrial function. WDFY3 (WD repeat and FYVE domain containing 3, also called ALFY) is involved in vesicle trafficking and membrane dynamics, especially autophagy. It is highly expressed in the central nervous system and disruption of the Drosophila homolog, bchs, disrupts axonal transport of endolysosomal vesicles [(Lim & Kraut 2009\)](https://paperpile.com/c/wQZsmN/ifSB) and yields brain degeneration [(Finley et al. 2003\)](https://paperpile.com/c/wQZsmN/kaM1). In humans, WDFY3 is a genetic risk factor for intellectual and developmental disabilities (IDD), microcephaly and neuropsychiatric disorders including schizophrenia, as reviewed in [(Dragich et al. 2016\)](https://paperpile.com/c/wQZsmN/HasK). Several other hits are implicated in autophagy, such as CIDEA and TMEM91’s family members, and mitochondrial function, such as SHPK [(Franceschi et al. 2022\)](https://paperpile.com/c/wQZsmN/qbIE) and SLC25A35, a mitochondrial Solute Carrier Family 25 protein family member widely expressed in the CNS [(Haitina et al. 2006\)](https://paperpile.com/c/wQZsmN/8AEY). CIDEA also transcriptionally regulates UCP1 (Mitochondrial Brown Fat Uncoupling Protein 1\) which is also a Solute Carrier Family 25 Member [(Jash et al. 2019\)](https://paperpile.com/c/wQZsmN/VNKv).  Other hits have associations to neurodevelopmental disorders, including RHOBTB2 [(Langhammer et al. 2023\)](https://paperpile.com/c/wQZsmN/OVUH) and CNOT3 [(Martin et al. 2019\)](https://paperpile.com/c/wQZsmN/l0py). Finally, ETS2 was identified as the causal gene for a non-coding variant in chromosome 21q22 [(Stankey et al. 2024\)](https://paperpile.com/c/wQZsmN/eJTQ). This is relevant because 21q22 is associated with Down’s Syndrome, which is associated with increased psychiatric/psychosis risk [(Rivelli et al. 2022\)](https://paperpile.com/c/wQZsmN/cSDl).

## **Discussion**

As reviewed in the introduction, the connection between mitochondrial structure/function and psychotic disorders is well-established from multiple streams of evidence. Here, we found in a cohort of 168 patients that those experiencing psychosis showed mitochondria less localized at the edge of fibroblasts. By querying existing large chemical and genetic perturbation datasets in cancer cell lines, we found that many genes and compounds that impact mitochondrial localization are implicated in brain function and psychiatric conditions.

Transport of mitochondria is tightly regulated and linked to human disorders [(Mandal & Drerup 2019\)](https://paperpile.com/c/wQZsmN/u8DO). Although the specific phenotype we observed, sometimes called perinuclear mitochondrial clustering, is relatively poorly understood, it may be associated with several other disorders [(Al-Mehdi et al. 2012\)](https://paperpile.com/c/wQZsmN/Vjd8). The aberrant transport of mitochondria has been linked to schizophrenia and bipolar disorder via several lines of evidence [(Flippo & Strack 2017; Scaini et al. 2021\)](https://paperpile.com/c/wQZsmN/Z0wI+mt7A). For example, the gene DISC1, Disrupted in Schizophrenia 1, is linked to schizophrenia, depression, bipolar disorder and autism [(Murphy & Millar 2017\)](https://paperpile.com/c/wQZsmN/hCSi), and mitochondrial dynamics in neurons were impacted by altering DISC1 levels [(Atkin et al. 2011\)](https://paperpile.com/c/wQZsmN/FNhu) and by large aggregates of DISC1 [(Atkin et al. 2012\)](https://paperpile.com/c/wQZsmN/EmhY). Mood stabilizers were found to impact proteins involved in mitochondrial transport in synaptoneurosomal preparations from rat pre-frontal cortex [(Corena-McLeod et al. 2013\)](https://paperpile.com/c/wQZsmN/PsQL). The *C. elegans* protein ANC-1, homolog of human genes (SYNE1/2) associated with various neuropsychiatric conditions, was found to promote localization of mitochondria to the base of the proximal axon [(Fischer et al. 2022\)](https://paperpile.com/c/wQZsmN/jf7w). Knockout of the schizophrenia-associated protein dysbindin yielded a significant decrease in mitochondrial density in distal axons, which impaired calcium dynamics [(Suh et al. 2021\)](https://paperpile.com/c/wQZsmN/XB9E). Together with our prior findings including autopsied brain [(Cataldo et al. 2010\)](https://paperpile.com/c/wQZsmN/xSLK), we believe there is sufficient evidence that the mitochondrial localization phenotype we observe in human patient fibroblasts here is also found in other brain-relevant cell types and in physiological structures *in vivo*.

We can only speculate as to the mechanism by which mitochondrial localization might impact neuron function and ultimately brain function. A recent study found that rather than producing ATP as is typical in most contexts, axonal mitochondria in cortical pyramidal neurons in mice actually consume ATP [(Hirabayashi et al. 2024\)](https://paperpile.com/c/wQZsmN/QMSV); this builds upon prior work showing that presynaptic release sites lacking mitochondria have a higher likelihood of neurotransmitter release than those with mitochondria [(Kwon et al. 2016; Vaccaro et al. 2017\)](https://paperpile.com/c/wQZsmN/EjFW+Sw5T). Our findings may be consistent with this: the lack of mitochondrial transport away from the nucleus that we observed here would imply that in psychosis, fewer mitochondria result in more ATP/energy being available in axons, reducing barriers to neurons firing. However, mitochondria produce numerous substrates necessary for cell function, including neurotransmission. These substrates may be lacking in terminals with lower levels of mitochondria.

This study has several important limitations. First, because of the challenges of obtaining consented patient samples and growing multiple patient cell samples, our cohort is limited to 168, spread across the conditions tested, and was collected in one clinical location with subjective diagnostic annotation. Still, this is larger than most cohorts published for other disorders and offers some mitigation of concerns about donor variability due factors such as genetics, sex, age, and comorbidities. Second, it is virtually impossible to obtain samples from patients who are untreated with medications, and most patients are on multiple medicines of varying mechanisms. Although the phenotypes we observe are those that persist in cells cultured through several passages and media changes and across patients taking different classes of medicines, we cannot rule out impacts by drugs the patients had been taking. Likewise, we cannot control for other unrecorded factors associated with these conditions, such as diet and exercise. Third, the study assessed a single cell type from patients, fibroblasts, because they are readily obtainable and robust in culture. As noted in the introduction, this strategy has been successful for other brain disorders, and other transport mechanisms have been found consistent across many cell types [(Goering et al. 2023\)](https://paperpile.com/c/wQZsmN/TnIf). Nevertheless, we have begun to explore mitochondrial localization in more physiologically relevant brain cells, specifically astrocytes derived from the fibroblasts used in the studies reported here \- preliminary results show altered mitochondrial localization consistent with that reported in this study; such reprogramming of the fibroblasts to induced pluripotent stem cells (iPSC) followed by induced differentiation to astrocytes is time-consuming but also has the advantage of resetting most epigenetic marks, thus reducing concerns about medication exposure or other environmental differences between patients and controls.

Finally, the use of this mitochondrial phenotype as a platform for drug discovery should be approached with caution, as we have previously noted more generally [(Chandrasekaran et al. 2020\)](https://paperpile.com/c/wQZsmN/G0Ap). Most screening strategies that translate findings from cell culture to the human organism are subject to caution; reversing disease-associated phenotypes poses some additional pitfalls. For instance, disease-associated phenotypes may reflect correlation with the patients’ problematic symptoms and not causation. In such cases, drugs reversing the phenotype may be ineffective or “even worse, the detected phenotype might reflect the cells’ attempt to mitigate the impact of the disease perturbation, such that drugs reversing the phenotype would aggravate the condition in patients” [(Chandrasekaran et al. 2020\)](https://paperpile.com/c/wQZsmN/G0Ap). That is, the observed phenotype may be a useful compensation for the abnormalities underlying illness. In that case, reversing the compensation would be anti-therapeutic, although given the tentative observation here of a spectrum of localization with psychosis on one end and depression on the other, medicines at either end might prove useful.

We found that several existing classes of medications used to treat psychosis were not associated with mitochondrial mislocalization. Optimistically, the mitochondrial localization phenotype could represent a novel mechanism by which psychosis might be remedied and is therefore worthy of future chemical screening. Still, there are several reasons why drugs that treat these conditions may not reverse the mitochondrial phenotype in patient fibroblasts nor cancer cell lines. Potential alternative explanations include: (a) the cell type (whether the cancer cell line used in pubic datasets stained for mitochondria studied here, or a particular patient’s fibroblasts that might be used in a future screen) might be missing key components necessary for proper response, especially the targets of the given drugs. Given that drugs do not work equally well across patients, we would expect heterogeneity in assay response as well, so a future screen would ideally test multiple patient cell lines. As noted earlier, testing more brain-relevant cell types would be ideal, albeit costly, though this may not be necessary for a primary drug screen for compounds working by this mitochondrial localization mechanism. (b) The timing may not be appropriate; existing psychiatric medications typically take 2-3 weeks to show relief of symptoms in patients, although it is also possible that this is only the case due to the complexity of full organ systems. (c) Drug metabolic processes that are only functional in the whole organism may be necessary for the impact of effective drugs in this context. (d) Drug dose, although tested across a range, may be unsuitable.

Future studies should evaluate the genes and compounds we identified for their impact on mitochondrial localization directly in a collection of patient cell lines, whether fibroblasts or iPSC-derived neural cells, to confirm mitochondrial phenotype reversal and bolster generalizability. It could also be valuable, albeit expensive, to physically screen compounds to find those impacting mitochondrial localization in fibroblasts from patients experiencing psychosis, rather than this study’s virtual query in unaffected patients’ cancer cell lines. In such a setting, where images are collected in a single well-controlled experiment, searching for compounds that reverse a machine-learning based disease phenotype is more feasible, rather than the single feature used here to facilitate virtual screening using publicly available data.

Overall, we found here that the mitochondrial phenotype we originally observed in a small cohort of bipolar patient fibroblasts and which was seen in our prior study of post mortem brain, is now consistently observed across a much larger group of patients experiencing the most common conditions associated with psychosis. We find this phenotype is induced by perturbation of genes associated with these conditions, and is influenced by drugs that impact pathways known to influence symptoms in human patients. Given the strong evidence in the literature for mitochondrial function disruption in these diseases, and given that the phenotype could be simplified to a single feature, the mitochondrial dispersion phenotype appears to be well suited for further virtual and physical screening to identify novel targets and therapeutic candidates.

## **Methods**

The protocol for cell culture, sample preparation, and imaging is described in Methods but also provided online (<https://github.com/carpenter-singh-lab/2024\_Haghighi\_Mito/blob/main/McleanCollectionFibroblastGrowthProtocol.md>).

## ***Primary cell lines***

Fibroblasts were derived as in [(McPhie et al. 2018\)](https://paperpile.com/c/wQZsmN/z4bc).  All subjects provided written informed consent, as approved by the McLean Hospital/Mass General Brigham (formerly Partners Healthcare) Institutional Review Board IRB\# 2007P002597.

Human fibroblast cell lines were grown as in our prior study [(Cataldo et al. 2010\)](https://paperpile.com/c/wQZsmN/xSLK). Three passages were done prior to plating onto glass coverslips for staining and imaging. Cells were plated at a density of 7.9 x 103 cells per cm2. MitoTracker-M7510, 125 nmol/L (Invitrogen Corporation, Carlsbad, CA), was used to visualize mitochondria according to manufacturer’s instructions, with a 30 minute live incubation. Cytoskeletal integrity was examined by immunocytochemistry using Phalloidin-488 (1:500) according to the manufacturer’s protocol and nuclei were visualized by Hoechst 33342 trihydrochloride (2 ug/ml). Two coverslips per line were mounted in Fluoro-Gel aqueous mounting media (Electron Microscopy Sciences, Hatfield, PA).

## ***Imaging***

Fifty fields were imaged at 400X in three independent channels on a Zeiss Axiovert  Observer 2.1 with a Colibri 2 LED illumination system. Image acquisition was done with Zeiss Zen 2.0 software. Single channel images were exported as high resolution TIFF files as both color and black and white images as well as one merged three-color image. Images were visually checked for focus, defined edges on each cellular element to be analyzed.

## ***Image analysis and feature engineering***

A set of images, obtained as described above, of nuclei, cytoskeleton, and mitochondria were processed by an image analysis pipeline to quantify the mitochondria within individual cells using CellProfiler 3 [(McQuin et al. 2018\)](https://paperpile.com/c/wQZsmN/KXEb) or CellProfiler 4 [(Stirling et al. 2021\)](https://paperpile.com/c/wQZsmN/uQ4D). The nucleus was segmented using a seed-based watershed algorithm on DAPI images. The cell objects were seeded from nuclei objects and were segmented on actin images or on mitochondria images and then expanded to actin images. We also applied filters to remove cells touching the border, with segmentation issues, and with intensity artifacts as described in the Github repository for the project.

The mitochondria were also segmented, by first applying the tubeness transformation (<https://www.longair.net/edinburgh/imagej/tubeness>), enhancing slender objects and denoising the image, then mitigating under-segmentation by applying an image of a grid to split very large mitochondrial objects into more typically-sized objects. The segmented mitochondrial “objects” of an individual cell were related to each other by overlap with the cytoskeleton object.

The mitochondria features for segmented cells were extracted in several ways. Most relevant to this study, features relating the position and density of mitochondria within a cell were created by calculating the intensity of mitochondria radially across either 12 or 16 bins, from the center of the cell to the cell edge, center of the cell nucleus to the cell edge, and edge of the cell nucleus to the cell edge. Second, the mitochondria image within each cell was skeletonized and the number of branchpoints, ends, and mean lengths was measured. Third, the number of mitochondria objects (after the grid-based splitting step) within each cell was counted. The CellProfiler pipelines for processing fibroblast data are available at the project’s github repo.

For displaying the example hit images for the LINCS dataset, intensities were rescaled relatively, for visualization, as follows: the MitoTracker image is raised to power of 0.4 then rescaled to the min/max for each image. Then, the red-green-blue merged image is made with MitoTracker and DNA as red and blue, respectively, and rescaled with a set range of .05-.7. Finally, cell outlines from CellProfiler are overlaid in white.

## ***Patient group classification using machine learning***

We performed patient category classification for various psychosis spectrum subgroups versus control using the patient-level morphological representations. Each patient has a bag of single cell samples and we form per-bag representations as the median of single cell samples representations in each bag when training a traditional ML classifier. We used a subset of mitochondria channel features extracted by CellProfiler for learning a logistic regression classifier. We used raw single cell mitochondria channel images for learning a CNN deep learning based classifier and we evaluated performance of each learned model through stratified K-fold cross validation (K=10). The area under the receiver operating characteristic curve (AUC-ROC) is reported for each binary classification as the measure of classification performance.

CellProfiler features extracted from the mitochondria channel form a set of 326 highly correlated features. This multicollinearity in our data could affect the feature importance exploration analysis. We handled multicollinearity in our data by forming subgroups of highly correlated features and selecting one feature from each cluster as the feature selection criteria. Subgroups of features were formed by application of hierarchical clustering on Pearson correlation of features and median of each cluster was selected as the representative feature for each subgroup. Supplementary Figure 5 shows the clustering dendrogram and feature subgroups. Y-axis threshold for forming clusters was selected to form a maximum number of clusters while keeping a Variance Inflation Factor (VIF) less than 10 for each feature in the final set. This resulted in a set of 45 representative features used for feature importance exploratory analysis. We note that while using this representative set degrades the classification performance results, it was a necessary step  for  feature interpretation analysis which is crucial in this study. Classification results reported in Figure 3 consider all of the set of mitochondrial features, however the feature importance analysis was based on the models learned from the representative set of features (Supplementary Figure 6). This analysis examined only the patient categories that had an AUC-ROC \> 0.6 even after using the representative set of features for classification. Features with an AUC-ROC\< 0.6 were considered  to be indistinguishable from the Control class based on this analysis.

## ***Feature interpretation and design of the MITO-SLOPE metric***

We then used the 45 representative features for the Feature importance analysis. This analysis is based on the logistic regression model’s coefficients and permutation importance; the top ten features from this analysis are reported in Supplementary Figure 6\. Model coefficients and permutation importance results reveal that the following categories of features have the most distinctive power across various patient categories versus control samples:

(a) A subset of texture measures in the nuclei and cytoplasm area of the mito channel.  
(b) A subset of features measuring the intensity of the mitochondria area across the cell.

The features in category (b) are a comprehensive set designed to capture the spatial patterns of the mitochondrial network across the cell and the potential shifts of this network toward the center or edge. These features are based on concentric rings drawn on the cell and can start from the cell center, nucleus center, or edge of the nuclei \- denoted by C (for Cell), CCN (for Cell from Center of Nucleus) and CEN (for Cell from Edge of Nucleus), respectively, in feature names). These bins continue to the edge of the cells. The following three submetrics are measured for each of the bin drawing conditions:

* FracAtD: Fraction of total stain in an object at a given radius.  
* MeanFrac: Mean fractional intensity at a given radius, calculated as the fraction of total intensity normalized by the fraction of pixels at a given radius.  
* RadialCV: Coefficient of variation of intensity within a ring, calculated across 8 slices.

By inspecting the set of distinctive features, we designed the *MITO-SLOPE* metric based on expert knowledge of image analysis and cell biology in the context of psychiatric conditions. This metric is derived from the full pattern of intensity changes across bins within each cell. Figure 4 shows this pattern from the nuclei edge to the cell edge across 12 bins defined in the cytoplasm area of a cell. To retain the benefit of single features while being robust to potential noise and artifacts, we developed MITO-SLOPE to capture the full pattern of change. The promise of this metric lies in its invariance to the number of bins, cell size, cell-to-nuclei size ratio, cell segmentation errors, and other sources of batch-to-batch variations. Also, it is not impacted by changes that occur to mitochondria upon certain fixation techniques [(Whelan & Bell 2015\)](https://paperpile.com/c/wQZsmN/ORmC). This metric measures the slope of the change in the normalized intensity values in each bin, focusing on the final bins close to the cell edge. Additionally, we focus on the final bins because  we could not extract the exact same set of bins in all of the compound and genetic datasets. In those datasets the bins defined for extracting features either start from the cell center (C for LINCS dataset) or from the nuclei center (CCN for the rest of genetic or chemical datasets).  Therefore capturing the pattern at the final bins makes the pattern more transferable across datasets used for virtual screening. “Final” here is defined below (“last extremum”).

### **Algorithm for calculation of MITO-SLOPE**

*Inputs:*

* A table containing single cell profiles of all the population with mito\_tubeness radial-distribution columns for MeanFrac across all radial bins. Radial bins are the 12 rings ordered from nucleus edge to cell edge (indices: RING-11 … RING).  
* Labels for each cell (“Control” vs. patient groups).

  *Output:*

* **MITO-SLOPE** (a scalar per subject).

  *STEPS:*

1. **Column-wise standardization (z-scoring)**  
   Standardize each feature so that each column of the table  has mean 0 and SD 1\. All subsequent steps use these z-scored values.

2. **Per-subject aggregation (MeanFrac)**  
    For each subject, average the z-scored MeanFrac radial-bin columns across that subject’s cells to obtain a per-subject profile; Isubject\[i\] for i \= 0 … B−1 (B \= 12).

3. **Control mean profile**  
    Compute the mean profile across all Control subjects; Icontrol-mean\[i\]

4. **Control differencing**  
    For each subject, form I\[i\] \=Isubject\[i\]-  Icontrol-mean\[i\]

5. **Smoothing**  
    If desired (required on datasets with noisy measurements), apply a light 1D smoothing to I. (we applied a Savitzky–Golay filter to ΔI using window\_length \= 5 and polyorder \= 3 (length preserved). This reduces noise while retaining local trends.)

6. **Last extremum**  
    Find the positions of the global maximum and global minimum of I.  
    Restrict to indices 0 … B−3 (exclude last 2 bins prone to segmentation errors); if neither extremum lies in this range, set MITO-SLOPE \= 0 (no valid last extremum).  
    Otherwise, take the later (rightmost) of the two extremum positions within 0 … B−3 as the “last extremum.”

7. **Slope to the edge**  
    Compute **MITO-SLOPE** as:

   MITO-SLOPE \= I \[B-1\]-I\[last extremum\] (B-1) \-(last extremum)

   Units: z-score change per bin.

   **Interpretation:**

   • MITO-SLOPE \> 0 → peripheral enrichment of mitochondrial signal relative to controls.

   • MITO-SLOPE \< 0 → central shift relative to controls.

## ***Analysis for virtual screens of chemical and genetic perturbation datasets***

We perform the following data processing and analysis on each of the chemical and genetic perturbation datasets to obtain a ranked list of perturbations that modulate the target metric corresponding to mitochondrial dispersion relative to its untreated or control state.

We form a set of features based on patient fibroblast data that have weak linear correlation with the target metric (filtering to less than a 0.3 absolute correlation coefficient with the target metric) and are also not distinctive across various groups of patients versus the control group (filtering to less than a certain threshold defined on the mutual information between the features and the class labels). We decrease redundancy in the set by reducing the features so that those in the filtered set have less than a 0.8 correlation coefficient with each other. We call this the *orthogonal* set of features and use it as a control set of features in which minimal variability is desired. This means we do not want the genetic or chemical perturbations to dramatically change the other dimensions of cell morphology while modulating the target metric. The threshold p-value is set based on multiple hypothesis-corrected (adjusted) p-values that therefore differ across datasets. The values of target\_bh\_corrected\_critical\_dict were:  
{'taorf': 0.00409,  
'lincs': 0.00761,  
'CDRP': 0.00924,  
'jump\_orf': 0.00198,  
'jump\_crispr': 0.007,  
'jump\_compound': 0.00365}

Note that while the extracted morphological feature sets are almost consistent across various chemical and genetic datasets, the set of features and channels in patient fibroblast data differ from those in the perturbation datasets. We derive the orthogonal set based on the fibroblast dataset and based on the overlap of those features with those available in the other datasets.

Perturbation datasets are described in the next section “Screened datasets”. Each perturbation within a batch exists in multiple wells across a variable number of replicate plates. Each well consists of multiple sites, and per-site profiles are derived by averaging features across all single cells available in that site. The MITO-SLOPE metric is then derived from per-site profiles of the RadialDistribution pattern.

To mitigate the effect of plate-to-plate variation in our statistical analysis, we perform a two-sample t-test on per-site MITO-SLOPE values for control versus each perturbation within each plate. We also compare the orthogonal to the target set of features for control versus each perturbation using Hotelling's T² test. After forming a set of per-plate t-statistics for each perturbation, we select the plate with the median value of the target metric and retain its statistics as the final phenotype strength properties for that perturbation.

We also derive Cohen's d value and standardized p-value to account for variable sample sizes when comparing each perturbation versus control samples. The final list of perturbations is ranked by Cohen's d values and filtered to include only significant differences according to the corrected p-value using Benjamini-Hochberg multiple hypothesis correction.

## ***Screened datasets***

The chemical and genetic perturbation datasets used for virtual screening have previously been published and described. The cpg number allows freely accessing the images and related data from the Cell Painting Gallery (<https://github.com/broadinstitute/cellpainting-gallery>). Briefly, the datasets  include:

* cpg0004-lincs (LINCS, sometimes called LINCS-Pilot1), which is 1,571 compounds across 6 doses in 5 replicates in A549 cells [(Way et al. 2022\)](https://paperpile.com/c/wQZsmN/tjlZ). Images can be viewed here <https://idr.github.io/idr0125-way-cellpainting/>  
* cpg0012-wawer-bioactivecompoundprofiling (CDRP, also called CDRP-BBBC047-Bray), which is 30,430 compounds in U2OS cells at a single dose each, and in 4 replicates per compound [(Bray et al. 2017; Wawer et al. 2014\)](https://paperpile.com/c/wQZsmN/C8pQ+hLSB). Images can be browsed at Image Data Resource as idr0016.  
* cpg0016-jump (JUMP) which is 116,000+ compounds and 15,000+ genes (CRISPR knockout and overexpression) profiled in U2OS cells [(Chandrasekaran et al. 2023; Chandrasekaran et al. 2024\)](https://paperpile.com/c/wQZsmN/FzUY+aU1B). Subsets of images can be browsed at third party websites mentioned therein.  
* cpg0017-rohban-pathways (TA-ORF, sometimes called TA-ORF-BBBC037-Rohban) which is 323 overexpressed genes in 5 replicates in U2OS cells [(Rohban et al. 2017\)](https://paperpile.com/c/wQZsmN/erlm). Images can be browsed at Image Data Resource as idr0033.

## ***Enrichment analysis***

We performed an enrichment analysis in Figure 5\. Virtual screen analysis of each of the genetic or chemical datasets determines the strength of the target phenotype for each perturbation with respect to the control population through t-test statistics. We sort this differential morphological signature list by the Cohen’s d values and inspect enrichment of any candidate gene/compound set towards the top or bottom of the overall set using the GSEA strategy [(Subramanian et al. 2005\)](https://paperpile.com/c/wQZsmN/aEdO). The degree of a candidate set being overrepresented towards the extremes of the list is evaluated by an enrichment score (ES) which represents the deviation from a uniform distribution of the candidate set  throughout the list. To check for statistical significance of a calculated enrichment score, p-values are calculated by forming a null ES distribution and permutation testing. We used blitzGSEA [(Lachmann et al. 2022\)](https://paperpile.com/c/wQZsmN/tYgH) for the enrichment analysis in Figure 5\. Enrichment scores normalized by the size of the gene set (NES) and false discovery rate (FDR) for the presented NES values are reported for each set under enrichment analysis. FDR values are calculated using the Benjamini-Hochberg procedure to correct for multiple hypothesis testing across all gene/compound sets being analyzed.

## **Data and Code Availability**

Raw fibroblast cell images and single cell morphological profiles extracted by CellProfiler are available at <http://doi.org/10.5281/zenodo.15390513>. The CellProfiler pipelines for extracting them and source code to reproduce and build upon the presented results are available at the project’s github repository ([https://github.com/carpenter-singh-lab/2025\_Haghighi\_Mito](https://github.com/carpenter-singh-lab/2025_Haghighi_Mito)). Several results files are located in the results subfolder there. We license the source code as BSD 3-Clause, and license the data, results, and figures as CC0 1.0.

## **Funding**

Funding for this study was provided by the National Institutes of Health (NIH MIRA R35 GM122547 to AEC), Merkin Institute Fellowship from the Broad Institute of MIT and Harvard (to AEC),  BroadIgnite (to AEC), a NARSAD Distinguished Investigator Award (to BMC), a Harvard-MIT Neurodiscovery program (to BMC), and funds from the Program for Neuropsychiatric Research at McLean Hospital (to BMC).

## **Acknowledgements**

We gratefully acknowledge input from other members of the Cohen, Cimini and Carpenter–Singh labs as this project developed. In particular, we thank Dr. Yu Han for running preliminary analyses and Dr. Rakesh Karmacharya for manually annotating drugs into mechanism-of-action categories.
